docs/developer/releasing.md

```diff
@@ -161,8 +218,31 @@ helm-charts' repo `charts/agent-operator/crds`.
 2. Update references of agent-operator app version in helm-charts pointing to release version and bump helm chart version.
 You can use this [pull-request](https://github.com/grafana/helm-charts/pull/1831) as reference.
 
+### Updating Homebrew
+[Homebrew](https://brew.sh/) is a package manager for MacOS. It installs binaries 
+(aka "[bottles](https://docs.brew.sh/Bottles)") from either the main Homebrew repository 
+or a third-party one (aka a "[tap](https://docs.brew.sh/Taps)").
+
+For every Agent release we need to update two Brew repositories:
+* [homebrew/homebrew-core](https://github.com/Homebrew/homebrew-core) is the main one which 
+Brew installations use by default
+* [grafana/homebrew-grafana](https://github.com/grafana/homebrew-grafana) is a "tap" - 
```

Comment:
Rather than documenting our custom tap, should we follow in Mimir's footsteps and deprecate our own Formula in favor of the official Homebrew core one? See grafana/homebrew-grafana#41.

Reply:
The custom tap is still on 31.0, so it's been effectively deprecated...I'll remove it from the instructions for now and in the unlikely event that we are unable to decommission it we could add it back.